base/build-static-ioc.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -ex
+
+IOC=$1
+
+cp /opt/epics/RELEASE configure/
+cat << EOF > configure/CONFIG_SITE.local
+STATIC_BUILD=YES
+FULL_STATIC_BUILD=YES
+USR_LDFLAGS = -static-pie
+EOF
+
+git config --global --add safe.directory $PWD
+
+make_skip() {
+  if [ -z "$SKIP_CLEAN" ]; then
+    make "$@"
+  fi
+}
+
+make_skip distclean
+make
+make_skip clean
+make -C iocBoot
+
+version=$(git describe)
+
+echo $version > $IOC-version
+echo ".git/" > /tmp/globs
```

Comment:
The necessary globs for building afc-epics-ioc correctly are still going to be determined.

Reply:
Has this been resolved?